Assets/Controllers/GameManager.cs

```diff
@@ -182,21 +194,27 @@ public Vector3 GetPawnPosition(Pawn pawn)
 
     public void EndTurn()
     {
-        int playerCount = players.Count;
+        int playerIndex = GetNextPlayerIndex();
+        ClearSelectedPawn();
+        SetActivePlayer(players[playerIndex]);
+        activePlayer.StartTurn();
+        GetFirstPawnForActivePlayerPositionAndFocusCamera();
+    }
 
-        int playerIndex = players.FindIndex( x => x == activePlayer);
+    private int GetNextPlayerIndex()
+    {
+        int playerIndex = players.FindIndex(x => x == activePlayer);
```

Comment:
Is there value in finding the index of any player? Another abstraction opportunity where you pass in a player (in this case `activePlayer`) and it returns an index.